hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/HederaWorldState.java

```diff
@@ -355,6 +368,10 @@ private void trackNewlyCreatedAccounts(
 
```

Comment:
Up there at L365 it looks like we're looking to see if the nonce has a special value to see if this is a token account or not.  That seems indirect so I'm wondering if there's something if there's a more direct approach to `isTokenAccount()`?  (In fact I'm wondering if `isTokenAccount()` or something similar should be exposed from `UpdateTrackingAccount`.)

Actually, now looking at `org.hyperledger.besu.evm.account.Account` there isn't much there to work with (there or at its superclass `AccountState`).  So looks like this might be the best we can do over here in the EVM side of things.